src/components/CredderRating/FeedCredderRating.js

```diff
@@ -58,10 +58,10 @@ const FeedCredderRating = ({
 
   const renderCredderRatingScore = () => {
     if (!score || score < 0) return 'n/a';
-    // return ${score}${<Text>{`%`}</Text>}
+    const parsedScore = Math.round(parseInt(score, 10));
     return (
       <Text>
-        {`${score}`}
+        {`${parsedScore}`}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for ensuring that the score is parsed as an integer and rounded before being displayed. However, it's important to handle potential exceptions that could occur during parsing. If `parseInt(score, 10)` fails due to invalid input, it will return `NaN`, which would then be passed to `Math.round()`. This could lead to unexpected behavior or crashes. To prevent this, you should add error handling around the parsing operation.

```diff
- const parsedScore = Math.round(parseInt(score, 10));
+ let parsedScore;
+ try {
+   parsedScore = Math.round(parseInt(score, 10));
+ } catch (error) {
+   console.error(`Error parsing score: ${score}`, error);
+   return 'n/a';
+ }
```
This way, if there's an error during parsing, we log the error and return 'n/a' as the score.


<!-- This is an auto-generated comment by OSS CodeRabbit -->